src/output.jl

```diff
@@ -145,15 +145,16 @@ end
 saveproblem(out::Output) = saveproblem(out.prob, out.path)
 
 """
-    savediagnostic(diag, diagname, filename)
+    savediagnostic(diagnostic, diagname, filename)
 
-Save diagnostics in `diag` to `filename`, labeled by `diagname`.
+Save `diagnostic` to `filename` under name `diagname`. Only the computed diagnostic
+is saved, that is, everything up to diagnostic's iteration `diagnostics.i`.
```

Comment:
`diagnostics.i` should be `diagnostic.i`?

Reply:
Yes. Good catch.